demos/browser/app/meeting/meeting.ts

```diff
@@ -474,6 +475,30 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver
     });
   }
 
+  getNearestMediaRegion(): void {
+    const supportedMediaRegions: Array<string> = ['ap-northeast-1', 'ap-southeast-1', 'ap-southeast-2', 'ca-central-1', 'eu-central-1', 'eu-north-1', 'eu-west-1', 'eu-west-2', 'eu-west-3', 'sa-east-1', 'us-east-1', 'us-east-2', 'us-west-1', 'us-west-2'];
```

Comment:
Is there a reason for this extra check on support regions? If there's new meeting region launched in the future and client code copied this snippet - they'll got errors when new regions is returned from nearest-media-region